docs/gettingstarted/index.rst

```diff
@@ -132,12 +129,8 @@ Getting started with Flocker
 
          Install the Flocker client on your Linux machine:
 
-         .. version-code-block:: console
-
-            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
-              virtualenv flocker-tutorial && \
-              flocker-tutorial/bin/pip install --upgrade pip && \
-              flocker-tutorial/bin/pip install --quiet https://clusterhq-archive.s3.amazonaws.com/python/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
+         .. task:: install_cli ubuntu-14.04
```

Comment:
(From similar https://github.com/ClusterHQ/flocker/pull/1133/files#r24321366):

> The previous scripts use `&&` to join the commands intentionally.  That pattern stops the command sequence on the first failure.

I'm not sure if that matters enough but it is worth considering.
Maybe using the tasks (for this and Homebrew) should be done in a separate issue.
And perhaps having `&&`s are good.
I'll leave the decision to you.